README.md

```diff
@@ -40,29 +40,42 @@ At this time pya is more suitable for offline rendering than realtime.
 
 ## Authors and Contributors
 
-* Thomas Hermann, Ambient Intelligence Group, Faculty of Technology, Bielefeld University (author and maintainer)
-* Jiajun Yang, Ambient Intelligence Group, Faculty of Technology, Bielefeld University (co-author)
-* Alexander Neumann, Neurocognitions and Action - Biomechanics, Bielefeld University
+* [Thomas](https://github.com/thomas-hermann) (author, maintainer)
+* [Jiajun](https://github.com/wiccy46) (co-author, maintainer)
+* [Alexander](https://github.com/aleneum) (maintainer)
 * Contributors will be acknowledged here, contributions are welcome.
 
 ## Installation
 
-<!-- **Disclaimer**: We are currently making sure that pya can be uploaded to PyPI, until then clone the master branch and from inside the pya directory install via `pip install -e .` -->
+`pya` requires `portaudio` to play and record audio. 
 
-**Note**: pya can be installed using **pip**. But pya uses PyAudio for audio playback and record, and PyAudio 0.2.11 has yet to fully support Python 3.7. So using pip install with Python 3.7 may encounter issues such as portaudio. Solutions are:
+1. Disclaimer: As of December 2022, the latest `pyaudio==0.2.12` is not fully available on Conda Forge. This means 
+this methods is currently not suitable, as it will install 0.2.11 instead. This will work on older version of Pya (< 0.5.1). 
 
-1. Anaconda can install non-python packages, so that the easiest way (if applicable) would be to 
+Anaconda can install non-python packages, so that the easiest way (if applicable) would be to 
 
     conda install pyaudio
 
-2. For Mac users, you can `brew install portaudio` beforehand. 
+
+2. For Mac users, `brew install portaudio`.
+  - For Apple ARM Chip, do the following instead: [Installation on ARM chip](https://stackoverflow.com/a/73166852/4930109)
+      - `brew install portaudio`
+      - `brew --prefix poraudio`
+      - Create .pydistutils.cfg in your home directory, `~/.pydistutils.cfg`, add:
+
+```
+[build_ext]
+include_dirs=<PATH FROM STEP 3>/include/
+library_dirs=<PATH FROM STEP 3>/lib/
+```
+      - `pip install -r requirements.txt`
 
 3. For Linux users, try `sudo apt-get install portaudio19-dev` or equivalent to your distro.
```

Comment:
### Using PIP (Linux)

Linux users should install portaudio before installing PyAudio via pip:

```
sudo apt-get install portaudio19-dev  # for debian/ubuntu
pip install pyaudio
```